examples/ifsmon.py

```diff
@@ -17,12 +17,13 @@
 logging.basicConfig(level=logging.DEBUG)
 
 variables = {"Amon" : ["tas","uas","vas"]}
-startdate = datetime.date(1990,1,1)
+startdate = datetime.date(1870,1,1)
 interval = relativedelta(months=1)
 #srcdir = os.path.dirname(os.path.abspath(ece2cmorlib.__file__))
 #datadir = os.path.join(srcdir,"test","test_data","ifsdata","3hr")
 mydir = os.path.dirname(os.path.abspath(__file__))
-datadir = os.path.join(mydir,"..","test","test_data","ifsdata","3hr")
+#datadir = os.path.join(mydir,"..","test","test_data","ifsdata","3hr")
+datadir = os.path.join("/Users/anthoni-p/EC-Earth/supermuc/runs/r1902-merge-new-components_impi/LNTS/run_i108_n0_micro_phase2_10years/output/ifs/001")
```

Comment:
We would rather have an example that works on the included test data.

Reply:
Apologies for these, I should have checked and undone any changes not related to the LPJ-Guess CMORization before making the pull request

Reply:
No problem, we fix the examples when we resolve issue #223